.github/workflows/readthedocs_pr.yml

```diff
@@ -0,0 +1,35 @@
+name: Read the Docs PR Preview
```

Comment:
From the RTD docs, it looks like this workflow is not needed if one connects the RTD app

Reply:
True. I removed this workflow, and it's still rendering the preview, which is good!